test/Elastic.Apm.AspNetCore.Tests/AspNetCoreDiagnosticListenerTest.cs

```diff
@@ -90,24 +89,24 @@ public async Task TestJsonBodyRetrievalOnRequestFailureInAspNetCore(bool useOnly
 				var client = Helper.GetClient(agent, _factory, useOnlyDiagnosticSource);
 
 				var body = "{\"id\" : \"1\"}";
-				var response = await client.PostAsync("api/Home/PostError", new StringContent(body, Encoding.UTF8, "application/json"));
+				await client.PostAsync("api/Home/PostError", new StringContent(body, Encoding.UTF8, "application/json"));
 
 				capturedPayload.Should().NotBeNull();
 
-				capturedPayload.Transactions.Should().ContainSingle();
+				capturedPayload?.Transactions.Should().ContainSingle();
```

Comment:
The null conditionals aren't needed here IMO, as a null reference will fail the test on [line 94](https://github.com/elastic/apm-agent-dotnet/pull/982/files#diff-7c522236e53c6b6553577390af5e0b16cc269c08530571ec49216b7017ebecd9R94)

Reply:
Same as in https://github.com/elastic/apm-agent-dotnet/pull/982#discussion_r515686044